mob.go

```diff
@@ -1205,11 +1280,62 @@ func getCachedChanges() string {
 
 func makeWipCommit(configuration Configuration) {
 	git("add", "--all")
-	gitWithoutEmptyStrings("commit", "--message", configuration.WipCommitMessage, configuration.gitHooksOption())
+	commitMessage := configuration.WipCommitMessage
+	if configuration.WriteLastModifiedFileInCommitMessage && areFilesModified() {
+		commitMessage += " lastFile:" + getPathOfLastModifiedFile()
+	}
+	gitWithoutEmptyStrings("commit", "--message", commitMessage, configuration.gitHooksOption())
 	sayInfoIndented(getChangesOfLastCommit())
 	sayInfoIndented(gitCommitHash())
 }
 
+// uses git status --short. To work properly files have to be staged.
+func getPathOfLastModifiedFile() string {
+	gitstatus := silentgit("status", "--short")
+	lines := strings.Split(gitstatus,"\n")
+	lastModifiedFilePath := ""
+	lastModifiedTime := time.Time{}
+	rootDir := gitRootDir()
+
+	debugInfo("Find Filepaths")
+	for _, line := range lines {
+		if strings.HasPrefix(line, "D") || strings.HasPrefix(line, "R"){
+			continue
+		}
+		filepath := ""
+		if strings.HasPrefix(line, "M")  {
+			filepath = strings.TrimPrefix(line, "M")
+		} else if strings.HasPrefix(line, "A") {
+			filepath = strings.TrimPrefix(line, "A")
+		}
+		filepath = strings.TrimSpace(filepath)
+		debugInfo(filepath)
+		info, err := os.Stat(rootDir + "/" + filepath)
+		if err != nil {
+			panic(err)
+		}
+		if info.ModTime().After(lastModifiedTime){
+			lastModifiedTime = info.ModTime()
```

Comment:
extract `info.ModTime()` to variable `currentLastModifiedTime` so not to call it twice.

Reply:
done.